frame/session/src/historical/mod.rs

```diff
@@ -236,7 +236,7 @@ impl<T: Config> ProvingTrie<T> {
 		let mut root = Default::default();
 
 		{
-			let mut trie = TrieDBMut::new(&mut db, &mut root);
+			let mut trie = TrieDBMutV0::new(&mut db, &mut root);
```

Comment:
Same as transaction root, I did not change the trie format for historical session.
Not sure if a comment is needed.

Reply:
So this will stay this way for ever?

Reply:
Yes, we can try to move it to V1, but then we need to pass/attach the state version with proofs (to be compatible with old proofs).
IIUC this put accountid in in memory trie, so with current existing account id, the public key is 32byte encoded so we actually may be able to use V1 transparently.
Yet it looks easier in the context of this PR to stay on the safe side (even if I doubt there is substrate chain with account id that encode to more than 32 byte).

Reply:
I thought V0 and V1 are both compatible when it comes to reading? (I know that we have here a TrieDBMut, everything we write here should then also be decodable by V1)

Reply:
The issue is for replaying the old blocks.

Reply:
Well it s build from grandpa. Still sounds better to me for replay-ability.
Thinking twice I will be tempted to switch (since the encoded content looks smaller than 33 bytes).
But that could be done in a different step.

Reply:
> Well it s build from grandpa/babe

Where?

This is runtime code. Where is that being done on the client side?